var/spack/repos/builtin/packages/migrate/package.py

```diff
@@ -23,11 +23,15 @@ class Migrate(AutotoolsPackage):
     depends_on('automake')
     depends_on('libtool')
     depends_on('m4')
```

Comment:
Can you switch these 4 to `type='build'`? They shouldn't be needed after installation.